### ISSUE TITLE
As a user interested in 78rpm data, I want the program.sourcecatalogue field to be included in the open data, so that I can select items from the 78rpm catalogue

### ISSUE THREAD

**Issue by mwigham** [beeldengeluid/beng-lod-server#252]
Tasks:
- [x] Check copyright status of this field and whether it is ok to publish
- [x] Choose an appropriate property
- [x] Consider how homogeneous this collection is and whether we actually want to publish it
- [ ] Consider if we want to modify the content of this field for better usability (e.g. 78rpm is "handelsplaten_registratie.mdb")
- [ ] Add it to the schema.org mapping
~~- [ ] Deploy the new mapping so that the field is available in ODL~~

NB: I have removed the deployment step as I think it is better to bundle our changes and choose a good moment to deploy to ODL

Acceptance criteria:
- [ ] The content of the program.sourcecatalogue field appears in the ODL open data

---
Context:
https://github.com/beeldengeluid/x-omgeving/issues/651

**Comment by mwigham** [comment]
Options for the property:
[schema:isPartOf](https://schema.org/isPartOf). NB: this requires us to generate an identifier for the 'collection' described in the field. 
[schema:category](https://schema.org/category). This is allowed to have a text value as well as a URL. 

I think that schema:category is better suited to the uncontrolled text in the sourcecatalogue field

**Comment by mwigham** [comment]
Feedback from Tim: the field can be published (no rights issues) but the values are messy. More importantly, he doubts if the current query accurately targets 78rpm records. I have contacted Dik Plattel to check this and to ask if there are other fields we can use. Based on the answer we receive, we need to decide if we find this collection sufficiently homogeneous and consistent to publish.

**Comment by mwigham** [comment]
Answer from Dik: 78rpm is indicated in PAM but this field is NOT harvested by DAAN. Tim is checking with Yvette about why this might be.

Suggested workaround from Willem: If we can't get the necessary field, we could request an export from PAM to get the list of IDs and we add the collection indication when importing to LD.

**Comment by mwigham** [comment]
Another option is to use the handelsplaten_registratie.mdb if we can adequately define and describe what is in there

**Comment by mwigham** [comment]
Answer from Yvette: the number of fields imported from PAM was limited due to fears of the import slowing DAAN. So only a limited subset was imported, which sadly does not include speed.

**Comment by mwigham** [comment]
Tim suggested asking Dik and other music experts. I have passed the question on to Dik, Harry and Greg

**Comment by mwigham** [comment]
In the meantime, I found that there is quite a large overlap between 'handelsplaten_registratie.mdb' en 'schellak'. So we could make approximately the same collection available defined as the 'schellak collection' 

Handelsplaten_registratie.mdb	6610
Schellak	6608
Schellak but not handelsplaten_registratie.mdb	19
Handelsplaten_registratie.mdb but not schellak	21
Both 6589

**Comment by mwigham** [comment]
'Handelsplaten_registratie.mdb' is in any case not synonymous with '78 rpm'. I found a carrier collection 'Audio: Pathe 78 toerenplaten'  with 7 items that are not in this database, and another 'Audio Handelsplaten 78tpm nederlands repertoire' with 9 items not in.

**Comment by mwigham** [comment]
Tim is happy for us to include assetItems.materiaalsoort in CC0 (NB: this field is filled in for 369336 items in the archive)

@wmelder @gb-beng what do you think, shall we go with a collection based on material sort rather than speed?

**Comment by mwigham** [comment]
We have chosen not to publish this collection at this time. We considered various options:
- an export of the 78 rpm material from PAM. This would be timeconsuming and would only provide a static snapshot of the collection, and the question is how complete the speed field is
- Using material type as an approximate proxy for 78rpm. We decided against this as it seems quite arbitrary
- releasing the Groef 78rpm collection. This is a nice collection for playability, but playability is not a high priority at present
- publishing a collection of Handelsplaten with persons linked. One of the key reasons for choosing 78rpm was that it has a high percentage of persons linked. Publishing a collection of Handelsplaten with persons would achieve the same goal. However, we decided against this, as it would be a very 'functional' collection, really only suitable for linked data geeks

We decided to stick to the MOZ collection for the time being. Once that has been completed, we can consult with stakeholders to see which (if any) of the above options would be a good choice for a second collection.
